hexchat/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: hexchat
+version: "2.12.1"
+summary: IRC client for X based on X-Chat 2
+description: |
+  HexChat is a graphical IRC client with a GTK+ GUI. Features include Python
+  and Perl scripting support, a plugin API, multiple server/channel windows,
+  spell checking, multiple authentication methods including SASL, and
+  customizable notifications. For more information on IRC, see
+  http://irchelp.org/.
+confinement: strict
+parts:
+  hexchat:
+    plugin: autotools 
+    source: git://github.com/hikiko/hexchat.git
```

Comment:
something picky that **I** usually do, change `git://` to `https://` because the protocol used in `git://` isn't secure